src/core/UserIconGroup/UserIconGroup.tsx

```diff
@@ -0,0 +1,121 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import cx from 'classnames';
+import { UserIcon } from '../UserIcon';
+import { CommonProps } from '../utils/props';
+
+export type UserIconGroupProps = {
+  /**
+   * UserIconGroup max number of icons unstacked
+   * @default 5
+   */
+  maxIcons?: number;
+  /**
+   * UserIconGroup is group needs to be stacked
+   * @default 'true'
+   */
+  stacked?: boolean;
```

Comment:
@mayank99 Added stacked?: boolean for unstacked version

Reply:
I updated and simplified the stories to focus on the props of `UserIconGroup` (instead of repeating `UserIcon` sizes).

Seems that `stacked: false` doesn't do anything:
![image](https://user-images.githubusercontent.com/9084735/124652565-40ccdf00-de6a-11eb-9565-48f2f247c2b3.png)

We need a new css class for it.